baseline/ica/score_generation/score_loader.py

```diff
@@ -80,7 +45,7 @@ def get_data(is_test_run: bool):
     return ica_df_n1_abs, ica_df_n3_abs, compositions_df_n1, compositions_df_n3
 
 
-def _get_data_for_norm(
+def _load_scores_for_norm(
```

Comment:
Load doesn't imply compute? At least IMO - since that can happen here

Reply:
`load_composition_df_for_sample` is a good example of loading

Reply:
Yeah agreed, but unsure what to call it instead.

Reply:
Anything saying "load scores" should be `get` or `compute` instead of load is my suggestion